Gopkg.lock

```diff
@@ -3,14 +3,14 @@
 
 [[projects]]
   branch = "master"
-  digest = "1:d7582b4af1b0b953ff2bb9573a50f787c7e1669cb148fb086a3d1c670a1ac955"
+  digest = "1:fc0802104acded1f48e4860a9f2db85b82b4a754fca9eae750ff4e8b8cdf2116"
```

Comment:
Need to regenerate with dep v0.4.1 to avoid lockfile churn.  Should just be installing the right version and running `dep ensure`